source/Core/BSP/Miniware/configuration.h

```diff
@@ -247,18 +249,14 @@
 #define TEMP_NTC
 #define I2C_SOFT_BUS_2 1
 #define SC7_ORI_FLIP
-#endif
-#endif
-
-#ifdef MODEL_TS101
-#define FLASH_LOGOADDR (0x08000000 + (126 * 1024))
-
-#else
-#define FLASH_LOGOADDR (0x08000000 + (62 * 1024))
-#endif
+#endif /* TS80P */
 
 #ifdef MODEL_TS101
+#define FLASH_LOGOADDR      (0x08000000 + (126 * 1024))
 #define SETTINGS_START_PAGE (0x08000000 + (127 * 1024))
 #else
-#define SETTINGS_START_PAGE (0x08000000 + (63 * 1024))
-#endif
\ No newline at end of file
+#define FLASH_LOGOADDR      (0x08000000 +  (62 * 1024))
+#define SETTINGS_START_PAGE (0x08000000 +  (63 * 1024))
+#endif /* TS101 */
+
+#endif /* CONFIGURATION_H_ */
```

Comment:
I may not understanding some particularities so correct me if I'm wrong but shouldn't `endif` for `include`-guard be in the very end of the file, right?

Reply:
yeah thats a mistake